### DISCUSSION TITLE
Using Alpine with handlebars

### DISCUSSION THREAD

**Discussion by alexdohmen** [alpinejs/alpine#1187] [1. Help] [answered]
Hey, I am new to alpine and looking for an alternative to backbonejs w/ handlebars. The only requirement being handlebars.

Did anyone try to use alpine with handlebars?

**Comment by HugoDF** [comment, marked as answer]
It shouldn't be a problem:
- Alpine doesn't use any interpolation syntax eg. `{{ }}` so there won't be a clash there (Alpine would use `x-text` for example)
- Alpine watches for new components being dynamically inserted and initialises them (if you're using handlebars client-side)

**Reply by alexdohmen** [reply]
ok, sounds promising

**Reply by HugoDF** [reply]
yep, Alpine was designed by Caleb to fit a niche for a simple way to add reactive JavaScript components to server-rendered applications (Laravel, Rails, Django etc) so handlebars (server-side) + Alpine should work great.

**Reply by alexdohmen** [reply]
will give it a try, if I succeed will add it here

**Reply by alexdohmen** [reply]
@HugoDF thx for your fast reply

**Comment by alexdohmen** [comment]
As promised: https://codepen.io/alexdohmen/pen/qBqwwYX
cool stuff
